asapdiscovery-docking/asapdiscovery/docking/workflows/cross_docking.py

```diff
@@ -81,9 +81,11 @@ def cross_docking_workflow(inputs: CrossDockingWorkflowInputs):
     """
 
     output_dir = inputs.output_dir
-    if output_dir.exists():
+    if output_dir.exists() and inputs.overwrite:
         rmtree(output_dir)
-    output_dir.mkdir()
+    else:
+        output_dir.mkdir(exist_ok=True, parents=True)
+
     logger = FileLogger(
```

Comment:
Would it be better to move the logger initialization before line 83, and log whether or not the directory has been overwritten or not? Either way, I think there should be something reported as what has actually be done.

Reply:
oh that's a good point!